crates/ra_syntax/src/ast/syntax.asdl

```diff
@@ -1,3 +1,37 @@
+binOp = BooleanOr(pipepipe)
```

Comment:
added token set generation to make possible generate builders for nodes like `BinExpr`